### PR TITLE
Stroke "Write your first post" in the writing flow

### PR DESCRIPTION
**Pull request by paulopmt1**
Related to 2216-gh-Automattic/dotcom-forge

Asked for desing input here: p1682461024801329-slack-CKZHG0QCR

## Proposed Changes

* Stroke "Write your first post" in the writing flow

image

## Testing Instructions

* Using an unlaunched site, access the start-writing flow: http://calypso.localhost:3000/setup/start-writing/launchpad?siteSlug={yourUnlaunchedSiteDomain}
* Ensure the user can't get back to the post by clicking at `Write your first post`:
* Switch to the normal 'write' flow: http://calypso.localhost:3000/setup/write/launchpad?siteSlug={yourUnlaunchedSiteDomain}
* Ensure user can click normally
* Ensure tests are passing with this command: `yarn run test-client:watch`

## Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [x] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

**Comment by paulopmt1** [comment]
Asked for input here: p1682461024801329-slack-CKZHG0QCR

Once we get approval, we'll move this PR forward.

File: client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts
Comment (DustyReagan) [inline client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts]: I think we want to use `isStartWritingFlow` rather than commandeering `isWriteFlow` since it's being used in a couple places.

**Comment by lupus2k** [comment]
I noticed the `Skip to dashboard` link at the bottom of the screen is not working. Should we create a new issue to fix this or are we already taking care of this?
image

**Review by lupus2k — APPROVED** [review]
All works as expected. Good job 🎉

File: client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts
Comment (paulopmt1) [reply client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts]: Good catch @DustyReagan, [changed it here](https://github.com/Automattic/wp-calypso/pull/76249/commits/e85b55b40e614cc8a44459f8782c4aa8ab3bdb07).

**Comment by paulopmt1** [comment]
> I noticed the `Skip to dashboard` link at the bottom of the screen is not working. Should we create a new issue to fix this or are we already taking care of this? image

Good catch @lupus2k, this was fixed [on Dusty's change](https://github.com/Automattic/wp-calypso/pull/76165). Thanks!

**Review by DustyReagan — APPROVED** [review]
LGTM!

**Pull request merged**
